packages/apps/docs/package.json

```diff
@@ -32,6 +32,7 @@
     "test:watch": "vitest"
   },
   "dependencies": {
+    "-": "^0.0.1",
```

Comment:
what's the use of this package and `D` both seems archived and not maintained anymore?